app/src/test/java/org/oppia/android/app/testing/administratorcontrols/AdministratorControlsFragmentTest.kt

```diff
@@ -101,82 +86,6 @@ class AdministratorControlsFragmentTest {
     ApplicationProvider.getApplicationContext<TestApplication>().inject(this)
   }
 
-  @Test
-  fun testAdministratorControlsFragment_clickEditProfile_checkSendingTheCorrectIntent() {
-    launch<AdministratorControlsActivity>(
-      createAdministratorControlsActivityIntent(
-        0
-      )
-    ).use {
-      testCoroutineDispatchers.runCurrent()
-      onView(withId(R.id.edit_profiles_text_view))
-        .perform(click())
-      intended(hasComponent(ProfileListActivity::class.java.name))
-    }
-  }
-
-  @Test
-  fun testAdministratorControlsFragment_clickAppVersion_checkSendingTheCorrectIntent() {
-    launch<AdministratorControlsActivity>(
-      createAdministratorControlsActivityIntent(
-        0
-      )
-    ).use {
-      testCoroutineDispatchers.runCurrent()
-      onView(withId(R.id.administrator_controls_list)).perform(
-        scrollToPosition<RecyclerView.ViewHolder>(
-          3
-        )
-      )
-      onView(withId(R.id.app_version_text_view)).perform(click())
-      intended(hasComponent(AppVersionActivity::class.java.name))
-    }
-  }
-
-  @Test
-  @Config(qualifiers = "sw600dp")
-  fun testAdministratorControlsFragment_clickEditProfile_checkLoadingTheCorrectFragment() {
```

Comment:
this test should not be removed from here. as we are testing the fragment here, it's not a UI test.

Reply:
Though the similar test added in the new test file `testAdministratorControlsFragment_clickEditProfile_checkLoadingTheCorrectFragment` sounds good to me, but we need to check if that UI test is only working on a tablet or it needs to work on mobile too or not?

Reply:
Can you please explain this a bit more: 

> but we need to check if that UI test is only working on a tablet or it needs to work on mobile too or not?

Reply:
@anandwana001 I want to bring this into your notice: Running `testAdministratorControlsFragment_clickEditProfile_checkLoadingTheCorrectFragment` is failing as the fragment is already in view and after performing a click action over the `Edit Profiles` button it is loading another fragment on top of it instead it should check if the fragment is already in view then do not re-instantiate it again. The same is happening if we click more than one time on the `App Version` button. It is getting matched with multiple views because of re-instantiating the same fragment over the previously instantiated one.

Reply:
This test is now handled both for tablet and mobile.